sdk/storage/azure-storage-common/inc/azure/storage/common/internal/xml_wrapper.hpp

```diff
@@ -37,44 +37,39 @@ namespace Azure { namespace Storage { namespace _internal {
 
   class XmlReader final {
   public:
-    explicit XmlReader(const char* data, size_t length);
```

Comment:
Our coding guidelines say that constructors should generally be marked explicit to ensure that they are not accidentally invoked (there are exceptions however where the implicit construction can be useful to improve code clarity).

Was there a need to allow constructing the XmlReader implicitly?

Reply:
No specific reason for removing `explicit`, except the rule that `explicit` has no effect for multi-argument constructors like here.

EDIT: I just read that this changed as per C++11: https://stackoverflow.com/a/39122237/22422288

So your coding guidelines have a good point. I'll add `explicit` back.